File: Cloud5S_API/DMS.Business/Filter/MD/VehicleFilter.cs
using DMS.BUSINESS.Filter.Common;

namespace DMS.BUSINESS.Filter.MD
{
    public class VehicleFilter : BaseFilter
    {
        public string TypeCode { get; set; }
    }

    public class VehicleFilterLite : BaseMdFilter
    {
    }

    public class VehicleExportFilter : BaseExportFilter
    {
        public string TypeCode { get; set; }
    }
}

File: Cloud5S_WEB/src/app/@filter/MD/vehicle.filter.ts
import {BaseFilter} from '../Common/base.filter';
import {EROLE_CODES} from 'src/app/utils/constant';

export class VehicleFilter extends BaseFilter {
  code: string = '';
  typeCode: string = '';
  unitCode: string = '';
  RoleCode: string = EROLE_CODES['LAI_XE'];

  driverUserName?: string = '';
}

export class AccountGetAllFilter {
  groupId: string = '';
  RoleCode: string = EROLE_CODES['LAI_XE'];
  KeyWord: string = '';
  isActive: boolean | string = true;
}
